dspace-api/src/main/java/org/dspace/authenticate/clarin/ShibHeaders.java

```diff
@@ -150,6 +151,10 @@ public void log_headers() {
     }
 
     private void initializeNetIdHeader() {
-        this.netIdHeader = configurationService.getProperty("authentication-shibboleth.netid-header");
+        this.netIdHeaders = configurationService.getArrayProperty("authentication-shibboleth.netid-header");
```

Comment:
do we need to call 
configurationService.getArrayProperty("authentication-shibboleth.netid-header"); 
twice?

Reply:
No, but update all classes, constructors would took some time. I'd like to survive this duplicate calling instead of spending more time optimizing the code.